### PR TITLE
Added examples where Switch as X can be used

### LINKED ISSUE #23452: Switch as X - suggested additional information
### Feedback

When you introduced the new "Switich as X" functionality I initially used the "add integration" button which guided me through the process. However, when I wanted to create another switch as x, I couldn't find how to do it without revisting the documentation and again using the manual add from the web site. Today, I spotted that this functionality is contained as a helper which is really useful. However, there is no reference to this in your "switch as x" page. I think it would be worth adding a brtief line or two to the documentation to inform users about this. Keep up the good work!

### URL

https://www.home-assistant.io/integrations/switch_as_x/

### Version

2022.7.5

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dannytsang**
## Proposed change
Added example where Switch as X can be used outside of the integrations page.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #23452

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/switch_as_x.markdown [hunk 1]
```diff
@@ -43,3 +43,16 @@ Switch entity:
...
 {% endconfiguration_basic %}
+
+## Alternatives
+Other ways to set a Switch as X include:
+ - [Helper](#helper)
+ - [Entity Properties](#entity-properties)
+
+### Helper
+{% my helpers title="Settings -> Devices & Services -> Helpers" %}
+![Helper example](/images/integrations/switch_as_x/Helper_Switch_as_X.png)
```
Comment (frenck) [inline source/_integrations/switch_as_x.markdown]: I'm not sure about this, this is already provided by the steps above?

image

File: source/_integrations/switch_as_x.markdown [hunk 2]
```diff
@@ -43,3 +43,16 @@ Switch entity:
...
+
+### Entity Properties
+{% my entities title="Settings -> Devices & Services -> Entities" %}
+![Entity properties exampl](/images/integrations/switch_as_x/Entities_Properties.png)
```
Comment (frenck) [inline source/_integrations/switch_as_x.markdown]: Let's just make a single chapter on: Configuring switch as X from an existing entity as an alternative method or something (so we don't need all these paragraphs at all).

Comment (dannytsang) [reply source/_integrations/switch_as_x.markdown]: I was struggling to fit the screenshots to the other ways of doing it but I will take another look later.

File: source/_integrations/switch_as_x.markdown (hunk 1 above)
Comment (dannytsang) [reply source/_integrations/switch_as_x.markdown]: Sorry my bad. I will remove it.

File: source/_integrations/switch_as_x.markdown (hunk 2 above)
Comment (dannytsang) [reply source/_integrations/switch_as_x.markdown]: Re-wrote this part along with the above comment to **Switch as X from existing entity**

File: source/_integrations/switch_as_x.markdown [hunk 3]
```diff
@@ -43,3 +43,7 @@ Switch entity:
 Type:
   description: The type of entity you want the switch to become.
 {% endconfiguration_basic %}
+
+## Switch as X from existing entity
+Where compatible, Switch as X will appear on entity properties. A list of entities can be found in {% my entities title="Settings -> Devices & Services -> Entities" %}.
+![Entity properties exampl](/images/integrations/switch_as_x/Entities_Properties.png)
```
Comment (frenck) [inline source/_integrations/switch_as_x.markdown]:
```suggestion
## Switch as X from the existing entity

Where compatible, Switch as X will appear on entity properties. A list of entities can be found in {% my entities title="Settings -> Devices & Services -> Entities" %}.

![Entity properties exampl](/images/integrations/switch_as_x/Entities_Properties.png)
```

**Review by frenck — APPROVED** [review]
Thanks, @dannytsang 👍

**Pull request merged**